diff --git a/python-mode.el b/python-mode.el
index 7d2451b..c4041b1 100644
--- a/python-mode.el
+++ b/python-mode.el
@@ -4297,6 +4297,7 @@ Returns `py-indent-offset'"
                          (py-end-of-statement)
                          (py-beginning-of-statement))
                 (current-indentation))))
+      (if (not firstindent) (setq firstindent 0)) ;; fix: firstindent nil error
       (when secondindent
         (when (eq 0 (abs (- secondindent firstindent)))
           (when (if (py-beginning-of-statement) (< (current-indentation) secondindent))
